.github/workflows/python-CI.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         run: rustup target add aarch64-apple-darwin
       - name: Setup Rust part of the project
         run: |
-          maturin build -i python --universal2 --out dist --no-sdist
```

Comment:
Why do we want to upload the `sdist` files?

Reply:
I think the wheels suffice

Reply:
This is because of the new version of maturin, the `no-sdist` flag disappeared in v0.13.
See https://www.maturin.rs/migration.html#from-012-to-013:
> maturin 0.13 replaced --no-sdist with the new --sdist option in maturin build command, source distributions are now only built when --sdist is specified.

Reply:
Oh, perfect then 😄 Thank you!